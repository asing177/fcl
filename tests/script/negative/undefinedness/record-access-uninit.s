type T {
  C1(int f1, bool f2, text f3);
  C2(int f1, bool f2);
  C3(int f1);
}

global T t;
global bool x;

@initial
go() {
  x = t.f2;
  terminate()
}
