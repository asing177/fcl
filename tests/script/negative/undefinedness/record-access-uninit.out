Error at Line 12, Column 8:
   Reference to invalid field 'f2'. The following fields are defined for every constructor:
      int f1